src/graphql/models/Badge.js

```diff
@@ -0,0 +1,31 @@
+import {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLNonNull,
+  GraphQLID,
+  GraphQLList,
+} from 'graphql';
+
+export default new GraphQLObjectType({
+  name: 'Badge',
+  fields: {
+    _id: { type: new GraphQLNonNull(GraphQLID) },
+    id: {
+      type: GraphQLID,
+      resolve: (badge) => badge._id,
+    },
```

Comment:
All the other models in `graphql/models` under do this:
```suggestion
    id: { type: new GraphQLNonNull(GraphQLID) },
```

This is because no matter the object is loaded via `docLoader` or by a search query, the utility function `processMeta` already turns `_id` to `id`.